week_1_1.py

```diff
@@ -0,0 +1,154 @@
+"""
+
+TASK 1
+
+Write a class to represent a Cash Register.
+You class should keep the state of price total and purchased items
+
+Below is a starter code:
+------------------------
+1. you can add new variables and function if you want to
+2. you will NEED to add accepted method parameters where required.
+For example, method add_item probably accepts some kind of an item?..
+3. You will need to write some examples of how your code works.
+
+"""
+
+
+class CashRegister:
+
+    def __init__(self):
+
+        self.total_items = []
+        self.total_price = 0
+        self.discount = 0
+        self.total_items_details = []
+
+    def add_item(self, item, price):
+
```

Comment:
This is nice and you've clearly thought about different circumstances (such as item already being in the receipt). The iterating through a `list` of `self.total_items_details` however is a bit clunky. Can you think of a way to implement this with a different data structure where you could access the right entry in `self.total_items_details` by name?